lms/templates/experiments/user_metadata.html

```diff
@@ -42,11 +43,19 @@
         'course_end',
         'dynamic_upgrade_deadline',
         'course_upgrade_deadline',
+        'audit_access_deadline',
 ):
     user_metadata[datekey] = (
         context.get(datekey).isoformat() if context.get(datekey) else None
     )
 
+for timedeltakey in (
+    'course_duration',
+):
```

Comment:
do you expect this list to grow beyond just the one key?

Reply:
Seems like it could happen, given that the list of date times has also grown several times.